test/upload-test-images.sh

```diff
@@ -17,14 +17,10 @@
 set -o errexit
 
 : ${1:?"Pass the directories with the test images as arguments"}
```

Comment:
The location of the test images is now fixed, you don't need arguments for this script anymore.